adapter/repository/in-memory-simple/src/main/resources/gtfs_spec.asciipb

```diff
@@ -429,7 +429,7 @@ csvspec: {
 csvspec: {
   name: "routes"
   filename: "routes.txt"
-  required: false  # required unless the feed is stops only.
+  required: true  # required unless the feed is stops only.
```

Comment:
Do we want to get rid of `unless the feed is stops only.`? This was a Google-thing - official GTFS spec doesn't support stop-only feeds.

Reply:
I think that we should, since that is a google specific thing. Will modify.